### PR TITLE
[4.0] Export of Banner Tracks not possible

### PR DESCRIPTION
**Pull request by Cherrypeach**
Pull Request for Issue #26818.
Export of Banner Tracks wasn't possible because the Modal Window was in the background.
### Summary of Changes
Fixed by changing the the Position of the Modal to Body.
For this, the Buttons that opens the Modal also changes the Position of the Modal.

### Testing Instructions
Go to Components -> Banners -> Tracks
Click on Export

### Actual result
image

### CODE REVIEW

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on d0cb1d1cc0de9f75772abf96c94808cc347b9e59<br /><br />Works now, thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by Quy** [comment]
Please do it like `Private Messages > My Settings`, then there is no need for this layout and workaround.

**Comment by shraddhaSankpal27** [comment]
I have tested this item :white_check_mark: successfully on d0cb1d1cc0de9f75772abf96c94808cc347b9e59<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by rajnish-dargan** [comment]
I have tested this item :white_check_mark: successfully on d0cb1d1cc0de9f75772abf96c94808cc347b9e59<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by rajnish-dargan** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by rajnish-dargan** [comment]
I have tested this item :white_check_mark: successfully on d0cb1d1cc0de9f75772abf96c94808cc347b9e59<br /><br />Functionality wise it is working but '?>' is coming on screen after applying patch<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by SharayuYadav** [comment]
I have tested this item :white_check_mark: successfully on d0cb1d1cc0de9f75772abf96c94808cc347b9e59<br /><br />Functionality wise it is working.
But '?>' is shown before the 'Export' button after applying the patch.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by Quiviro** [comment]
I have tested this item :white_check_mark: successfully on 9776218aa4db5972763de6681eb1f6541f62e1ae<br /><br />The modal window shows in the front
#PBF19es<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by Quy** [comment]
Please test PR #27091 for an alternate fix.

**Comment by Quy** [comment]
Closing in favor of PR #27091. Thank you for your contribution.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by Quy** [comment]
Please update per https://github.com/joomla/joomla-cms/pull/27091#issuecomment-557913646<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

**Comment by jwaisner** [comment]
@cherrypeach Please see update request per [#27091 (comment)](https://github.com/joomla/joomla-cms/pull/27091#issuecomment-557913646 "").<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26848">issues.joomla.org/tracker/joomla-cms/26848</a>.</sub>

File: layouts/joomla/toolbar/modal.php [hunk 1]
```diff
@@ -32,12 +32,14 @@
 ?>
 
 <!-- Render the button -->
-<joomla-toolbar-button<?php echo $id; ?> onclick="document.getElementById('modal_<?php echo $selector; ?>').open()" class="<?php echo $class; ?>" data-toggle="modal">
+<joomla-toolbar-button<?php echo $id; ?> onclick="document.getElementById('modal_<?php echo $selector; ?>').open();
+	$(document.getElementById('modal_<?php echo $selector; ?>')).appendTo('body');"
```
Comment (wilsonge) [inline layouts/joomla/toolbar/modal.php:36]:
```suggestion
	document.body.appendChild(document.getElementById('modal_<?php echo $selector; ?>'));"
```

Not at a machine with a setup but I think this should remove the jQuery dependency from this.

**Comment by Quy** [comment]
Please test PR #28472

**Pull request closed without merging**